src/main/java/games/strategy/triplea/ui/EditPanel.java

```diff
@@ -341,6 +344,7 @@ public void actionPerformed(final ActionEvent event) {
             advance.add((TechAdvance) selection);
           }
         } catch (final Exception e) {
+          ClientLogger.logQuietly(e);
```

Comment:
I have a feeling this catch block was added to compensate for an NPE or some other unexpected unchecked exception.  Logging it so we can possibly debug it in the future.